From 1b6bb498571bd17918ad1373de201d59c1724a64 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Thu, 14 Mar 2019 03:55:59 -0700
Subject: [PATCH 0573/1852] dt-bindings: display: xlnx: Add max width and
 height as mandatory properties

commit bf5063d5c7084b433e93d7535ef338aa4a1a6c9a from
https://github.com/Xilinx/linux-xlnx.git

The patch adds xlnx,max-width and xlnx,max-height mandatory device tree
properties. This breaks backward compatibility.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt
index 312c82d7ee25..6e83436ffaaa 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt
@@ -29,6 +29,12 @@ Required properties:
 - clock-names: Must contain "aclk_ctrl" and "aclk_axis" in same order as clocks
   listed in clocks property.
 
+- xlnx,max-width: Maximum number of pixels in a line.
+  Valid range from 64 to 8192.
+
+- xlnx,max-height: Maximum number of lines in a frame.
+  Valid range from 64 to 4320.
+
 Example:
 	scaler@a0040000 {
 		compatible = "xlnx,vpss-scaler";
@@ -39,4 +45,6 @@ Example:
 		xlnx,pix-per-clk = <2>;
 		clock-names = "aclk_ctrl", "aclk_axis";
 		clocks = <&misc_clk_0>, <&misc_clk_1>;
+		xlnx,max-width = <3840>;
+		xlnx,max-height = <2160>;
 	}
-- 
2.31.1

